src/main.ts

```diff
@@ -54,6 +55,7 @@ export async function run(argv = process.argv.slice(2), options?: Interfaces.Loa
     argv = argv.filter(arg => !getHelpFlagAdditions(config).includes(arg))
     const Help = await loadHelpClass(config)
     const help = new Help(config, config.pjson.helpOptions)
```

Comment:
I did not want to add another parameter to the constructor, so capturing original argv and then setting it directly o n the Help class instance.